scripts/run_all_tests.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+set -ex
+
+shout "Setting up"
+
+mkdir bin
```

Comment:
IIUC, this will create a `bin` directory under `cif` directory where we're cloning ci-firewall code, right? Same applies to next few steps where we're using `pwd`. This is by design, right?

Reply:
This has nothing to do with cloning of ci-firewall. This is the script that runs jenkins side.

It will create a directory called `bin` in whatever is the directory when the script runs. On jenkins this will be the workspace. 
I will add more comments to make this clear

Reply:
Is this for latter retrive of artifacts, why are we not using the default gobin ?

Reply:
Mostly to prevent overlap between multiple scripts. East test instance gets its own GOBIN